tests/pythonMeep.bash

```diff
@@ -0,0 +1,8 @@
+#/bin/bash
+
+#
+# test for successful loading of python meep module
+#
+cd ../python
+ln -f -s .libs/_meep.so
+python -c 'import meep'
```

Comment:
The advantage of doing it this way is that it clarifies that `pythonMeep` is just one of 20 single-word phrases in the `TESTS` definition in `tests/Makefile.am`. I'm concerned that having multiple lines of raw shell code in `Makefile.am` would make it harder to read, and would single out this test from all the others (why is the body of this one in `Makefile.am` instead of a separate file like all others?). I agree that we have here a case of shell invoking shell, but in my view it's worth it to put the `pythonMeep` test on equal footing with the other tests.

Or maybe you are envisioning a way of incorporating the test into `Makefile.am` that would be more transparent than what I am envisioning?